lib/galaxy/model/__init__.py

```diff
@@ -10555,3 +10614,6 @@ def receive_init(target, args, kwargs):
         if obj:
             add_object_to_object_session(target, obj)
             return  # Once is enough.
+
+
+JobStateSummary = namedtuple("JobStateSummary", enum_values(Job.states) + ["all_jobs"])  # type: ignore[misc]  # Ref https://github.com/python/mypy/issues/848#issuecomment-255237167
```

Comment:
```suggestion
JobStateSummary = NamedTuple("JobStateSummary", [(value, int) for value in enum_values(Job.states)] + [("all_jobs", int)])  # type: ignore[misc]  # Ref https://github.com/python/mypy/issues/848#issuecomment-255237167
```

Still dynamic, so not testable with mypy, but a bit more explicit w.r.t typing.